drivers/adc/ad713x/ad713x.c

```diff
@@ -549,6 +549,8 @@ int32_t ad713x_init(struct ad713x_dev **device,
 	if(NO_OS_IS_ERR_VALUE(ret))
 		goto error_gpio;
 
+	no_os_mdelay(10);
```

Comment:
just curious, is this delay related to the datasheet specifications? 

Also i'd move it in the gpio_init function.

Reply:
Yes, this delay has been mentioned in the datashseet for the PDN functionality.